DxDispatch/src/model/JsonParsers.cpp

```diff
@@ -367,7 +367,8 @@ bool ParseBoolField(const rapidjson::Value& object, std::string_view fieldName,
 
 half_float::half ParseFloat16(const rapidjson::Value& value)
 {
-    return ParseFloatingPointNumber<half_float::half>(value);
+    auto parsedValue = ParseFloatingPointNumber<float>(value);
+    return half_float::half(parsedValue);
```

Comment:
Interesting. Not saying we shouldn't have it, but what was the motivation?

Reply:
The template function did not like the half logic for bad casting.  Seems like we need an assignment instead of cast anyways since the exponent bits need to be shifted.